blackbox/docs/sql/administration/user_defined_functions.txt

```diff
@@ -80,6 +80,21 @@ For example, if you wanted to name two ``geo_point`` arguments ``start_point`` a
    Argument names are used for query documentation purposes only. You cannot
    reference arguments by name in the function body.
 
+Optionally, a schema for the function can be specified. If no schema is
+specified, the function will be created in the current session schema.
+
```

Comment:
Optionally, you can specify a schema for the function. If you omit the schema, the current session schema is used.